pre-commit.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Perhaps we move this into `tools` to prevent clutter on the project root?

Reply:
Good point. Done.